assets/style/sass/style.css

```diff
@@ -0,0 +1,663 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  font-family: "Red Hat Display", sans-serif;
+  min-height: 100vh;
+  line-height: 26px;
+  width: 100%;
+}
+
+.container {
+  margin: 0 auto;
+  overflow: hidden;
+}
+.container .card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  max-width: 375px;
+}
+@media only screen and (min-width: 480px) {
+  .container .card {
+    width: 100%;
+    max-width: 768px;
+  }
+}
+@media only screen and (min-width: 820px) {
+  .container .card {
+    width: 100%;
+    max-width: 1440px;
+  }
+}
+@media only screen and (min-width: 1440px) {
+  .container .card {
+    width: 100%;
+    max-width: 1440px;
+  }
+}
+
+header .right-side, header .left-side {
+  position: relative;
+  display: flex;
+  justify-content: space-around;
+  min-height: 188px;
+  height: 100%;
+  gap: 16px;
+  align-items: flex-start;
+}
+
+header {
+  display: grid;
+  grid-template-columns: 1fr repeat(2, minmax(100px, 1fr)) 1fr;
+  grid-template-rows: auto;
+  width: 100%;
+  align-items: center;
+}
```

Comment:
Nestovanje nije dobro u donjem delu fajla. Sve klase topa .logo, .left-side ... pa cak i responsive definicije za to trbaju biti nestovane ovde a ne da ih pises kao u obicnom css-u